### PR TITLE
compatibility with django 1.10

### PR DESCRIPTION
**Pull request by dvandok**
Please consider this for inclusion, in addition to https://github.com/cobbler/cobbler/pull/1854.
This should be pep8 clean by now.

### CODE REVIEW

**Comment by dvandok** [comment]
H'm, now I'm confused? Did you already integrate the changes I made earlier? What else did you mean in https://github.com/cobbler/cobbler/pull/1854#issuecomment-351171589?

**Comment by dvandok** [comment]
...And my local pep8 checker didn't catch some errors caught by `make qa`.

File: cobbler/web/urls.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 # from cobbler_web.contrib import admin
 # admin.autodiscover()
 
-urlpatterns = [
+urlpatterns = patterns(
```
Comment (kwizart) [inline cobbler/web/urls.py]: I still don't get why this change ? Any django urls.py created after django-admin startproject foo with not have this patterns('', ...)

Comment (dvandok) [reply cobbler/web/urls.py]: I can't recall for sure, but this was probably the result of trial-and-error. I don't claim to know much about django (or cobbler...)

Comment (eb4x) [reply cobbler/web/urls.py]: Yep, this is wrong. The correct way has already been committed.

**Comment by kwizart** [comment]
I think others PR was merged to fix this support.
Please re-open if there is anything missing)
(there is still a need to backport for cobbler28 - keeping django 1.6 support would be nice)

**Comment by dvandok** [comment]
I'm closing this as the fix is already in.
